.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+---
+repos:
+  # Formats import order
+  - repo: https://github.com/pycqa/isort
+    rev: 5.12.0
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: ["--profile", "black", "--filter-files"]
+
+  #Code formatter for both python files and jupyter notebooks
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
```

Comment:
I noticed this version might be a bit behind the latest. Consider running `pre-commit autoupdate` to help keep this file up to date with the latest versions of these checks (unless there's good reason not to do so).

Reply:
Updated! Thank you for pointing this out!